synapse/event_auth.py

```diff
@@ -385,36 +385,36 @@ def _check_size_limits(event: "EventBase") -> None:
 
     # Whole PDU check
     if len(encode_canonical_json(event.get_pdu_json())) > MAX_PDU_SIZE:
-        raise EventSizeError("event too large", strict=True)
+        raise EventSizeError("event too large", unpersistable=True)
```

Comment:
I think the docstring above here needs `s/strict/unpersistable/g`

Reply:
The same applies to `EventSizeError.__init__` and other comments elsewhere

Reply:
well spotted, thanks